frontend/src/features/documents/documentUpload/DocumentUploadContainer.tsx

```diff
@@ -48,10 +61,11 @@ export const DocumentUploadContainer: React.FunctionComponent<IDocumentUploadCon
     <DocumentUploadView
       documentTypes={documentTypes}
       isLoading={retrieveDocumentMetadataLoading || uploadDocumentLoading}
-      mayanMetadata={[]}
+      mayanMetadata={documentTypeMetadata}
       onDocumentTypeChange={onDocumentTypeChange}
       onUploadDocument={onUploadDocument}
       onCancel={props.onCancel}
+      documentMetadata={[]}
```

Comment:
If this is always an empty array, is it necessary to pass it as a prop?

Reply:
removed